signals/generators/ios/conversion.py

```diff
@@ -16,6 +16,20 @@
     Field.VIDEO: "NSURL*"
 }
 
+SWIFT_DATA_TYPES = {
```

Comment:
It feels like these lists should move to the templates file and the methods in here get new generic names like `get_template_data_type`